src/main/java/mate/academy/cinema/controllers/MovieController.java

```diff
@@ -0,0 +1,43 @@
+package mate.academy.cinema.controllers;
```

Comment:
```suggestion
package mate.academy.cinema.controller;
```
пекеджі називаємо в однині